src/datasets/other/addons_daily/reference.md

```diff
@@ -4,16 +4,16 @@ This ETL code produces daily aggregates of Firefox extensions. It supports the b
 
 ## Introduction:
 
-The addons_daily dataset serves as the central hub for all Firefox extension related questions. This includes questions regarding browser performance, user engagement, click through rates, etc. Each row in the table represents a unique addon, and each column is a unique metric.
+The `addons_daily` dataset serves as the central hub for all Firefox extension related questions. This includes questions regarding browser performance, user engagement, click through rates, etc. Each row in the table represents a unique addon, and each column is a unique metric.
 
 ### Contents
-Prior to construction of this dataset, extension related data lived in several different sources. Addons_daily has combined metrics aggregated from several sources, including raw pings, telemetry data, and google analytics data.
+Prior to construction of this dataset, extension related data lived in several different sources. `Addons_daily` has combined metrics aggregated from several sources, including raw pings, telemetry data, and google analytics data.
 
 ### Accessing the Data
 The data is stored as a parquet table in S3 [here](s3://net-mozaws-prod-us-west-2-pipeline-analysis/bmiroglio/addons_daily_test/)
```

Comment:
Its better to just display the path rather than link it, mainly for copy+paste reasons. Also the path for the productionized dataset is `s3://telemetry-parquet/addons_daily/v1/`.